webpack.config.js

```diff
@@ -1,4 +1,25 @@
 const WebpackErrorNotificationPlugin = require('webpack-error-notification')
+const webpack = require('webpack')
+
+let apiHost
+
+const setupAPI = () => {
+  switch ( process.env.NODE_ENV ) {
+    case 'production':
+      apiHost = "'http://localhost:1337'"
+      break
+    case 'test':
+      apiHost = "'http://localhost:1337'"
+      break
+    case 'development':
+      apiHost = "'http://localhost:1337'"
+      break
+    default:
+      apiHost = "'http://localhost:1337'"
+  }
+}
+
+setupAPI()
```

Comment:
About whether to take it out of the file or leave it, lets leave it for now and we can refactor later if it gets bigger.